docs/reference/core-concepts.md

```diff
@@ -7,3 +7,73 @@ title: Great Expectations Core Concepts
 # Data Docs
 
 # Profiling
+
+Great Expectations provides a mechanism to automatically generate expectations, using a feature called a **Profiler**. A Profiler builds an Expectation Suite from one or more Data Assets. It may also validates the data against the newly-generated Expectation Suite to return a Validation Result. There are several Profilers included with Great Expectations.
+
+A Profiler makes it possible to quickly create a starting point for generating expectations about a Dataset. For example, during the init flow, Great Expectations currently uses the UserConfigurableProfiler to demonstrate important features of Expectations by creating and validating an Expectation Suite that has several different kinds of expectations built from a small sample of data. A Profiler is also critical to generating the Expectation Suites used during profiling.
+
+## Rule-based Profilers
+
+**Rule-based profilers** allow users to provide a highly configurable specification which is composed of **Rules** to use in order to build an **Expectation Suite** by profiling existing data.
```

Comment:
As noted by @alexsherstinsky in one of my earlier PR's, can we standardize the spelling of `Rule-Based Profilers` throughout for consistency purposes?